fastai/vision/learner.py

```diff
@@ -112,6 +112,43 @@ def plot_top_losses(self, k, largest=True, figsize=(12,12)):
             cl = int(cl)
             im.show(ax=axes.flat[i], title=
                 f'{classes[self.pred_class[idx]]}/{classes[cl]} / {self.losses[idx]:.2f} / {self.probs[idx][cl]:.2f}')
+            
+    def plot_multi_top_losses(self, samples:int=3, figsz:Tuple[int,int]=(8,8)):
+        "Show images in `top_losses` along with their prediction, actual, loss, and probability of predicted class in a multilabeled dataset."
+        if samples >20:
+            print("Max 20 samples")
+            return
+        losses, idxs = self.top_losses(self.data.c)
+        infolist, ordlosses_idxs, mismatches_idxs, mismatches, losses_mismatches, mismatchescontainer = [],[],[],[],[],[]                                                      
+        truthlabels=np.asarray(self.y_true, dtype=int) 
+        classes_ids=[k for k in enumerate(self.data.classes)]
+        predclass=np.asarray(self.pred_class)
+        for i, pred in enumerate(predclass):
+            where_truth=np.nonzero((truthlabels[i]>0))[0]
+            mismatch=np.all(pred!=where_truth)
+            if mismatch: 
+                mismatches_idxs.append(i)
+                losses_mismatches.append((losses[i][pred],i))
+            infotup=(i, pred, where_truth, losses[i][pred], np.round(self.probs[i], decimals=3)[pred], mismatch)
+            infolist.append(infotup)
+        mismatches = self.data.valid_ds[mismatches_idxs]
+        ordlosses=sorted(losses_mismatches, key = lambda x: x[0], reverse=True)
+        for w in ordlosses: ordlosses_idxs.append(w[1])
+        mismatches_ordered_byloss=self.data.valid_ds[ordlosses_idxs]
+        print(mismatches)
+        mismatchescontainer=[]
```

Comment:
What is the purpose of this variable? I don't see it use after.

Reply:
I put it there to act as a container for misclassified items, so you have them ready in case you want to do a few epochs just upon them as suggested by Jeremy in Lesson 3. If you deem it appropriate, I can duly document that feature in the documentation notebook, or remove it in case you think it not to be useful.
Thanks!

Reply:
It's not returned or stored so how do you use it?

Reply:
You are right: I forgot to take it into account in the refactored version. Sorry.
Now, the method accepts an optional boolean parameter (save_misclassified). If True, it returns a list containing the misclassified samples (by default it's False).
I updated the documentation nb accordingly.
Thanks!

Reply:
Note: could you run `show_doc()` upon my method in the documentation nb, so it shows the signature? I couldn't manage to get this done. Thanks!!